src/lib/util/Config.ts

```diff
@@ -88,6 +88,18 @@ const DEFAULT_CONFIG: ConfigContent = {
 			],
 			disabled: false
 		},
+		{
+			name: ENGINE.ESLINT_CUSTOM,
+			targetPatterns: [
+				// Reduce default to be only what the other eslint engines can do (i.e. js and ts files).
+				// See https://github.com/forcedotcom/sfdx-scanner/issues/1146
+				// If user wants other files, then they update this inside of their ~/.sfdx-scanner/Config.json file
+				"**/*.js",
+				"**/*.ts",
+				"!**/node_modules/**",
+				"!**/bower_components/**"
+			]
+		},
```

Comment:
@stephen-carter-at-sf  - All other engines define a "disabled" field? Is that false by default? Can we still add it for consistency?

Reply:
This custom engine is enabled if you supply the --eslintconfig flag. The other engines use the "disabled" flag to help determine if the engine should run by default or not. Therefore it isn't needed here and wouldn't be used by the  isEnabled method.